shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -159,6 +164,7 @@ const CodeEditor = createClass({
 			autoCloseTags     : true,
 			styleActiveLine   : true,
 			showTrailingSpace : false,
+			theme             : this.props.editorTheme
```

Comment:
Here's another place where we are using a different name.  Can we make sure to consistently name any of these related variables "editorTheme" so as not to confuse with "brew themes"? Should make tracing the variable down the props chain a little easier too.

Reply:
Pretty sure that particular instance is required to set the CodeMirror internal theme parameter. The rest of `codeEditor.jsx` uses `editorTheme`. I'll check through the other files for consistency.